### DISCUSSION TITLE
Dịch `Feature Scalling`

### DISCUSSION THREAD

**Discussion by quangnhat185** [mlbvn/handson-ml2-vn#16] [Terminology] [answered]
`Feature Scalling` thì quá quen thuộc rồi nhưng mình không biết dịch ra tiếng Việt là gì cho phù hợp. Mn thấy dịch là `Tỉ lệ hóa Đặc Trưng` có ổn không?

Ngoài ra trong phần này còn mention đến hai phương pháp `min-max scaling` và `standardization`. Mình nghĩ nên giữ nguyên, không biết ý mn thế nào?

cc: @mlbvn/handson-ml2

**Comment by minhduc0711** [comment]
mình dò lại repo cũ thì thấy có mấy từ này
* `feature scaling` -> `biến đổi tỉ lệ đặc trưng`
* `standardization` -> `chuẩn tắc hóa` (do `standard normal distribution` hay được dịch là `phân phối chuẩn tắc`)
* `min-max scaling` dịch hết ra thì có vẻ hơi dài, vậy để là `biến đổi tỉ lệ min-max` chăng?

**Comment by cuongvng** [comment]
em vote `tỉ lệ hoá đặc trưng`, `chuẩn tắc hoá`, `tỉ lệ hoá min-max`.

**Reply by tiepvupsu** [reply]
Từ `hoá` trong `tỉ lệ hoá` nghe không đúng, mình không biết giải thích sao nhưng `tỉ lệ hoá` nghe rất khác `scaling`.

Nếu dịch là `co giãn đặc trưng` thì sao?

**Comment by quangnhat185** [comment, marked as answer]
Vậy mình làm cái poll mn vào vote nhé:

Dịch `feature scaling` và `min-max scaling`:

:+1:  : `co giãn đặc trưng`, `co giãn min-max`
:heart:  : `tỉ lệ hóa đặc trưng`, `tỉ lệ hóa min-max`

cc: @mlbvn/handson-ml2

**Reply by minhduc0711** [reply]
`co dãn` hay `co giãn` nhỉ :v

**Reply by thanhcsf** [reply]
@minhduc0711 co giãn em ạ

**Reply by cuongvng** [reply]
em tra thấy cả 2 đều đúng :v

**Reply by thanhcsf** [reply]
Nó giống như kỹ thuật/kĩ thuật hay hi sinh/hy sinh; dù sao cá nhân anh thấy co giãn dùng nhiều hơn..
